File: 1. Beginner/1005/1005.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`);
    val a = scanner.nextDouble();
    val b = scanner.nextDouble();
    println("MEDIA = %.5f".format((a * 3.5 + b * 7.5) / 11))
}

File: 1. Beginner/1005/1005.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(), let a = Float(inputA), let b = Float(inputB) {
    let r = (a * 3.5 + b * 7.5) / 11.0
    print(String(format: "MEDIA = %.5f",  r))
}